### PR TITLE
Various UX tweaks and front end fixes.

### LINKED ISSUE #225: Long URLs escape the post box in UI
We need a way to wrap them or truncate them.

### PR DESCRIPTION
**Pull request by TkTech**
- Add a nginx.dev.conf which is used by the dockerfile when django is in debug mode. This fixes issues with static assets not being available unless collectstatic is run even in debug mode.

Fixes #225

### CODE REVIEW

**Comment by TkTech** [comment]
Tweaks the behaviour of links in post boxes to match the hashtags (no underline, underline on hover) and switches their color to the Takahe highlight color to better standout as there were comments on the contrast being too low.

image

**Comment by TkTech** [comment]
Ultra-compact view on small screens:
image

**Comment by TkTech** [comment]
Moved action buttons to the bottom, which gives us a lot more room to work with on screens small and large, and is more typical of what you'd see on Twitter or Mastodon.

image

**Comment by TkTech** [comment]
Posts now show the total # of boosts and likes next to the action button.

Caveat, clicking the button doesn't immediately refresh the count. I don't really want to put the effort in until #231 is closed.

image

@andrewgodwin this is ready for review.

**Comment by whatSocks** [comment]
@TkTech I can't tell from the screenshots but when I'm testing on mobile (so an actual small screen) I sometimes want to see what version I'm on but it's hard to for me to see if the page I'm on even shows this information. Could this be displayed in the right side menu or someplace more accessible?

File: docker/nginx.conf [hunk 1]
```diff
@@ -9,7 +9,7 @@ http {
 
     include /etc/nginx/mime.types;
 
-    proxy_cache_path /cache/nginx levels=1:2 keys_zone=takahe:20m inactive=14d max_size=__CACHESIZE__;
+    proxy_cache_path /cache/nginx levels=1:2 keys_zone=takahe:20m inactive=14d max_size=1g;
```
Comment (andrewgodwin) [inline docker/nginx.conf:12]: Why did you remove the dynamic cache size replacement? This should be configurable.

File: docker/nginx.dev.conf [hunk 2]
```diff
@@ -0,0 +1,98 @@
+daemon off;
```
Comment (andrewgodwin) [inline docker/nginx.dev.conf:1]: Is the purpose of this entire file to remove /static/? If so, did you try `try_files` first, where the main static alias tries the local file first and then goes to proxy_pass?

**Review by andrewgodwin — COMMENTED** [review]
I have some questions about the nginx changes :)

As for the actions being at the bottom - I was trying to avoid using that vertical space, but I guess we don't have much option at some point. Lots more vertical space than horizontal.

File: docker/nginx.dev.conf (hunk 2 above)
Comment (TkTech) [reply docker/nginx.dev.conf:1]: I did not, not super familiar with nginx. And yes, it's to remove /static/

**Review by andrewgodwin — APPROVED** [review]
